strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.yaml

```diff
@@ -2,7 +2,7 @@ configuration:
   instanceName: strongbox
   version: ${project.version}
   revision: ${strongbox.revision}
-  baseUrl: http://localhost:48080/
+  baseUrl: ~
```

Comment:
What does `~` mean?

Reply:
It means [`null`](https://yaml.org/refcard.html)

Reply:
So, do we actually need this to be specified in the file?

Reply:
We don't support optional configuration fields at this point - so yes.

Reply:
Thanks for enlightening me!
:)